.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,38 @@
+---
+name: '🐛 Bug Report'
+about: Something isn't working
+---
+
+## Overview
+
+...
+
+## Consuming repo
+
+> What repo were you working in when this issue occurred?
+
+...
+
+## Labels
+
+- [ ] Add the `Type: Bug` label to this issue.
+
+### Area
+
+- [ ] Add any relevant `Area: *` labels to this issue
+
+### Scope
+
+- Is this issue related to a specific package?
+
+  - [ ] Tag it with the `Package: *` label.
+
+- Is it related to a new package?
+  - [ ] Tag it with the `New package` Label
+
+---
+
+## Checklist
```

Comment:
All these extra `Checklist` sections seem like duplications to the sectioned checklists above. We should stick with one or the other, imo.